TcUnit-Runner/Program.cs

```diff
@@ -351,6 +401,18 @@ static void DisplayHelp(OptionSet p)
             p.WriteOptionDescriptions(Console.Out);
         }
 
+        /// <summary>
+        /// Using the Timeout option the user may specify the longest time that the process 
+        /// of this application is allowed to run. Sometimes (on low RAM machines), the
+        /// DTE build process will hang and the only way to get out of this situation is
+        /// to kill the process.
+        /// </summary>
+        static private void KillProcess(Object source, System.Timers.ElapsedEventArgs e)
+        {
+            log.Error ("ERROR: timeout occured, killing process ...");
+            Environment.Exit(Constants.RETURN_TWINCAT_VERSION_NOT_FOUND);
```

Comment:
Why return TwinCAT-version not found for timeout? Maybe use a specific error code for that instead?